advisories/github-reviewed/2023/07/GHSA-cf7p-gm2m-833m/GHSA-cf7p-gm2m-833m.json

```diff
@@ -17,17 +17,12 @@
         "ecosystem": "PyPI",
         "name": "cryptography"
       },
-      "ecosystem_specific": {
-        "affected_functions": [
-          "cryptography.hazmat.primitives.serialization.ssh.SSHCertificateBuilder.sign"
-        ]
-      },
```

Comment:
I didn't meant to remove these lines and I cannot edit the PR online. Could you please re-add the lines?